recipes/Diamond/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: diamond
+  version: "0.7.9"
+
+source:
+  git_url: https://github.com/bbuchfink/diamond.git
+  git_rev: v0.7.9
+
+build:
+  number: 0
+
+test:
+  commands:
+    - diamond -h
+
+about:
+  summary: a BLAST-compatible local aligner for mapping protein and translated DNA query sequences against a protein reference database
+  home: http://github.com/bbuchfink/diamond
+  license: Copyright (c) 2014, University of Tuebingen
```

Comment:
The license is actually new BSD (https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_.28.22Revised_BSD_License.22.2C_.22New_BSD_License.22.2C_or_.22Modified_BSD_License.22.29)